shared/output/cloud/compute_small/test/google/cloud/compute/v1/addresses_rest_test.rb

```diff
@@ -93,27 +93,32 @@ def test_aggregated_list
 
         # Use hash object
         client.aggregated_list({ filter: filter, include_all_scopes: include_all_scopes, max_results: max_results, order_by: order_by, page_token: page_token, project: project, return_partial_success: return_partial_success }) do |_result, response|
-          assert_equal http_response, response
+          assert_kind_of ::Gapic::Rest::TransportOperation, response
+        assert_equal http_response, response.underlying_op
```

Comment:
Why is this indentation incorrect? If we can't figure this out quickly, then maybe just drop the `assert_kind_of` line and just change the `assert_equal`.

Reply:
no idea what is going on there. removed the extra for now.